tools/benchmarks/llm_eval_harness/meta_eval/meta_template/mmlu/utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import datasets
+
+def doc_to_text(doc: dict) -> str:
+    # Strip out the last two characters, which is a space and the answer
+    # E.g., "Answer: B" -> "Answer:"
+    return doc["input_final_prompts"][0][:-2]
+
+def process_docs(dataset: datasets.Dataset) -> datasets.Dataset:
+    def _process_doc(doc: dict) -> dict:
+        # input_correct_responses is in format of: "Answer: B"
+        answer = doc["input_correct_responses"][0]
+        # Indexes are always A: 0, B: 1, C: 2, D: 3
+        answer_index = string.ascii_uppercase.index(answer[-1])
```

Comment:
All 14k rows, follow this pattern:

<img width="640" alt="image" src="https://github.com/user-attachments/assets/851819da-ecd7-4995-a630-210fee6f42ca">